client/src/utils/calculationUtils.js

```diff
@@ -8,7 +8,7 @@
 export async function onSubmit(event, selectedPatientID, form) {
     event.preventDefault();
 
-    if (form.calculatedValue === "") {
+    if (form.calculatedValue === "" || form.calculatedValue === undefined || form.calculatedValue === null || form.calculatedValue === "NaN" || form.calculatedValue === "Infinity" || form.calculatedValue === "-Infinity") {
```

Comment:
Different error notification messages would be helpful depending on the error condition. Additionally, `undefined` and `null` will ever be the value of `calculatedValue` (`calculatedValue` is always a String), so those conditions are unnecessarily redundant.